pkgs/applications/networking/p2p/vuze/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchsvn, jdk, jre, ant, swt, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "vuze-${version}";
+  version = "5750";
+
+  src = fetchsvn {
+    url = "http://svn.vuze.com/public/client/tags/RELEASE_${version}";
+    sha256 = "07w6ipyiy8hi88d6yxbbf3vkv26mj7dcz9yr8141hb2ig03v0h0p";
+  };
+
+  buildInputs = [ makeWrapper jdk ant ];
+
+  buildPhase = "ant";
+
+  installPhase = ''
+    install -D dist/Vuze_0000-00.jar $out/share/java/Vuze_${version}-00.jar
+    makeWrapper ${jre}/bin/java $out/bin/vuze \
+      --add-flags "-Xmx256m -Djava.library.path=${swt}/lib -cp $out/share/java/Vuze_${version}-00.jar:${swt}/jars/swt.jar org.gudy.azureus2.ui.swt.Main"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Torrent client";
+    homepage = http://www.vuze.com;
+    license = licenses.free;
```

Comment:
This is not [free](https://www.gnu.org/philosophy/free-sw.en.html), please set the license to `unfree`.

See section 6.8 of the [terms of service](http://www.vuze.com/corp/terms):

> 6.8 Vuze Application License Grant.
> 
> (a)Single Copy Software License.  Subject to these Terms, Vuze grants you a non-exclusive, non-sublicensable, revocable as specified below, and non-transferable license to install and use one copy of the Vuze Application on one personal computer solely for your personal use in connection with the Vuze Platform as set forth in these Terms.  Except as expressly set forth herein, these Terms grant you no rights in the intellectual property of Vuze or any other party.  If you violate the terms of this license, the foregoing license will automatically terminate, and Vuze may in its sole discretion terminate these Terms, including without limitation your right to access and use the Vuze Platform.
> 
> (b)License Restrictions.  Except as explicitly permitted by these Terms, you represent and warrant that you shall not, and that you shall not permit or assist others to: (i) install or use the Vuze Application except under the license terms stated above; (ii) modify, translate, reverse engineer, decompile, disassemble (except to the extent that this restriction is expressly prohibited by law) or create derivative works based upon the Vuze Application; (iii) copy or publicly display or perform the Vuze Application; (iv) rent, lease, loan, sublicense, resell or otherwise transfer rights to or make any commercial use of the Vuze Application; (v) remove any proprietary notices or labels on the Vuze Application; or (vi) use the Vuze Application in any manner that violates any applicable law or regulation, including without limitation any third party copyright or other intellectual property or proprietary right.  You agree to immediately notify Vuze of any activities, by you or others, in violation of this or any other provision of these Terms.
> (c)Copies.  You may make one copy of the Vuze Application in machine-readable form for backup purposes only; provided that the backup copy must include all copyright or other proprietary notices contained in the original.
> 
> (d)Upgrades.  Vuze may, in its sole discretion, require you to perform, or automatically perform, updates, modifications, reinstallations, or the downloading of additional software or patches to the Vuze Application ("Updates").  By installing, running, or using the Vuze Application, you agree to permit or to perform all such Updates.  Each Update will be deemed a part of the Vuze Application and will be subject to these terms.

Reply:
unfree or unfreeRedisributable?

Reply:
> You may make one copy of the Vuze Application in machine-readable form for backup purposes only; provided that the backup copy must include all copyright or other proprietary notices contained in the original.

Makes me think unfree is the safest option.

Reply:
Agreed with joachifm :+1: